### PR TITLE
chore: add helm chart to project

### PR DESCRIPTION
**Pull request by AnguIar**

### CODE REVIEW

**Review by asafMasa — APPROVED** [review]

File: config/custom-environment-variables.json [hunk 1]
```diff
@@ -15,8 +15,10 @@
     "password": "DB_PASSWORD",
     "database": "DB_NAME",
     "schema": "DB_SCHEMA",
-    "logging": "DB_LOGGING",
-    "synchronize": "DB_SYNCHRONIZE",
+    "logging": {
```
Comment (cl-shaharshavit-rnd) [inline config/custom-environment-variables.json:18]: revert - db logging isnt bolean.
the possibale values includes "true" and "false" but also "query", "error", "schema", "info", "log", "all"

File: helm/values.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+global:
+  cloudProvider: {}
+  tracing: {}
+  metrics: {}
+  environment: {}
+  db:
```
Comment (cl-shaharshavit-rnd) [inline helm/values.yaml:6]: move to "rasterCommon", eventually every shared config that isnt in the boilerplate should be there.

File: helm/values.yaml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  protocol: "TCP"
+  openApiFilPath: "./openapi3.yaml"
+  logLevel: "warn"
+  tracing:
+    enabled: false
+    url: "http://localhost:55681/v1/trace"
+  metrics:
+    enabled: false
+    url: "http://localhost:55681/v1/metrics"
+  db:
```
Comment (cl-shaharshavit-rnd) [inline helm/values.yaml:78]: move to "rasterCommon". 
maybe apart from schema and logging? (for your consideration)

**Review by cl-shaharshavit-rnd — CHANGES_REQUESTED** [review]

File: config/custom-environment-variables.json [hunk 4]
```diff
@@ -15,12 +15,11 @@
...
     "schema": "DB_SCHEMA",
-    "logging": "DB_LOGGING",
-    "synchronize": "DB_SYNCHRONIZE",
-    "enableSslAuth": {
-      "__name": "DB_SSL_ENABLE",
+    "logging": {
+      "__name": "DB_LOGGING",
       "__format": "boolean"
     },
+    "enableSslAuth": "DB_SSL_ENABLE",
```
Comment (cl-shaharshavit-rnd) [inline config/custom-environment-variables.json:22]: db sll enable is boolean

**Review by cl-shaharshavit-rnd — CHANGES_REQUESTED** [review]

**Pull request merged**